platform/security/sts/security-sts-bstdelegationhandler/src/main/java/org/codice/ddf/security/delegation/BSTDelegationHandler.java

```diff
@@ -30,19 +28,13 @@
  */
 public class BSTDelegationHandler implements TokenDelegationHandler {
```

Comment:
❓ Is this still needed?

Reply:
I'm not sure that's why I just left it

Reply:
Wo. Looks like the two implementing classes we have for the `TokenDelegationHandler` interface are `BSTDelegationHandler` and `x509DelegationHandler`, which both do the same thing.

They both allow delegation if the given token parameters have a `BinarySecurityToken` on them. What that _actually_ means; I have no idea 🤷‍♂. But it sounds like this class might be able to be combined with the other class / deleted.

Reply:
Also, most of the code in `X509DelegationHandler` is commented out 😆